### ISSUE TITLE
Support for React Native 0.68.0

### ISSUE THREAD

**Issue by eliaslecomte** [ammarahm-ed/react-native-mmkv-storage#230]
**Describe the bug**
Hi, I'm trying to upgrade to React Native 0.68.0 (fom 0.66.x) and am bumping into a c++ error:

```
> Task :react-native-mmkv-storage:configureCMakeDebug FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':react-native-mmkv-storage:configureCMakeDebug'.
> C/C++: /Users/elias.lecomte/Projects/projectx/node_modules/react-native-mmkv-storage/android/src/main/rnmmkv/CMakeLists.txt debug|armeabi-v7a : CMake Error: The following variables are used in this project, but they are set to NOTFOUND.
  Please set them or make sure they are set and tested correctly in the CMake files:
  REACT_NATIVE_JNI_LIB
      linked by target "rnmmkv" in directory /Users/elias.lecomte/Projects/projectx/node_modules/react-native-mmkv-storage/android/src/main/rnmmkv
```

**To Reproduce**
1. Upgrade to React Native 0.68.0
2. Build your app (cd android && ./gradlew assembleDebug)

**Platform Information:**
 - OS: Android
 - React Native Version: 0.68.0
 - Library Version: 0.6.12

**Additional context**
We are also using Expo and react-native-reanimated but didn't have any issue before.

**Comment by eliaslecomte** [comment]
I've tried editing the CMakeLists.txt but couldn't figure out how to fix this.

**Comment by ammarahm-ed** [comment]
@eliaslecomte have you used the master branch? Try it. It works with RN v0.68

**Comment by focux** [comment]
I'm having the same issue. Just upgraded from RN 0.65 to 0.68. I'm also on the last version `0.6.12`. Any idea what could be wrong? I'm not using Expo but I'm using `react-native-reanimated` as the OP is.

**Comment by ammarahm-ed** [comment]
@focux Can you try the master branch and see if that works? 

```
npm install https://github.com/ammarahm-ed/react-native-mmkv-storage
```

**Comment by eliaslecomte** [comment]
> @eliaslecomte have you used the master branch? Try it. It works with RN v0.68

That is working ❤️, thanks!

I'm currently pinned on the latest master commit:
`"react-native-mmkv-storage": "git+https://github.com/ammarahm-ed/react-native-mmkv-storage.git#49ecfe1f9930049aa5a5dd235322cbf31d9b178e",`

All looks fine, if I come accross something strange, I will let you know!

**Comment by eliaslecomte** [comment]
Am I correct that I now also can remove:
```
packagingOptions {
    pickFirst '**/*.so'
}
```

(Added because of https://github.com/ammarahm-ed/react-native-mmkv-storage/issues/160)

**Comment by ammarahm-ed** [comment]
> Am I correct that I now also can remove:
> ```
> packagingOptions {
>     pickFirst '**/*.so'
> }
> ```
> 
> (Added because of https://github.com/ammarahm-ed/react-native-mmkv-storage/issues/160)

I think the you can remove this.

**Comment by focux** [comment]
It worked for me as well. Thank you for your help!

**Comment by OsamaAburabie** [comment]
@ammarahm-ed works for me as well thanks dude.

**Comment by focux** [comment]
So, it works for me when building for debugging but when I do a release build I'm getting the same error. Any idea?

**Comment by ammarahm-ed** [comment]
Which command are you using for release. Are you building the release build locally or in CI?

**Comment by focux** [comment]
Currently, I'm building locally using Fastlane and the command that is used to build the app is:
``gradlew clean app:assembleRelease -p android/``

**Comment by ammarahm-ed** [comment]
> Currently, I'm building locally using Fastlane and the command that is used to build the app is:
> ``gradlew clean app:assembleRelease -p android/``
> 

Try without gradlew clean. Just build for release directly and see if that works?

**Comment by focux** [comment]
That did it! No idea why It doesn't work with `gradlew clean` but thank you for helping me figure it out.

**Comment by ammarahm-ed** [comment]
> That did it! No idea why It doesn't work with `gradlew clean` but thank you for helping me figure it out.

You can still use gradlew clean. Just run both commands separately. As:

`gradlew clean && gradlew ...`. 

I will fix this before releasing next version soon.  Probably in a couple of days.

**Comment by kbwo** [comment]
I have tried `gradlew clean`, but still getting the same error on gradle sync.
React Native 0.80 and `"react-native-mmkv-storage": "github:ammarahm-ed/react-native-mmkv-storage#master"`(commit: 49ecfe1f9930049aa5a5dd235322cbf31d9b178e)

**Comment by ammarahm-ed** [comment]
Fix released on v0.7.0
